From 3e3e4f354ee21e6e7242bf2c302d33473a291471 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Tue, 7 May 2019 19:34:21 +0300
Subject: [PATCH 29/29] dts: Revert eMMC "compatible" string

Set the "compatible" property for USDHC3 to "fsl,imx7d-usdhc". With this value
eMMC storage works stable.

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
index 9854cf5087de..fa6ca04653ba 100644
--- a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
+++ b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dts
@@ -785,6 +785,7 @@
 };
 
 &usdhc3 {
+	compatible = "fsl,imx8mq-usdhc\0fsl,imx7d-usdhc";
 	pinctrl-names = "default", "state_100mhz", "state_200mhz";
 	pinctrl-0 = <&pinctrl_usdhc3>;
 	pinctrl-1 = <&pinctrl_usdhc3_100mhz>;
-- 
2.11.0

